netbout-web/src/test/java/com/netbout/dynamo/DyMessagesITCase.java

```diff
@@ -153,4 +158,32 @@ public void codeMarkdownInMessages() throws Exception {
             Matchers.equalTo("    4 leading spaces retained")
         );
     }
+    
+    /**
+     * DyMessages can throw {@code HttpException} if invoke {@code unread()}
+     * for user which is not in a bout.
+     * @throws URISyntaxException If there is some problem inside
+     * @throws IOException If there is some problem inside
+     */
+    @Test
+    public void exceptionIfBoutNotFound() throws Exception {
+        final String name = "testunread";
+        final Aliases aliases =
+            new DyBase().user(new URN("urn:test:75066")).aliases();
```

Comment:
@2686747 this (`DyBase` constructor) should still go in a try-with-resources to not leak anything.